sdk/objectanchors/Azure.MixedReality.ObjectAnchors.Conversion/src/Customizations/Models/AssetConversionConfiguration.cs

```diff
@@ -19,8 +19,9 @@ public partial class AssetConversionConfiguration
         /// </summary>
         /// <param name="gravity">Gravity vector with respect to object's nominal position.</param>
         /// <param name="scale">Scale of transformation of asset units into meter space.</param>
-        internal AssetConversionConfiguration(System.Numerics.Vector3 gravity, float scale)
-            : this(new Vector3(gravity), scale)
+        /// <param name="disableDetectScaleUnits">Whether or not disable automatic detection of FBX scale units.</param>
+        internal AssetConversionConfiguration(System.Numerics.Vector3 gravity, float scale, bool disableDetectScaleUnits = false)
```

Comment:
We don't need two internal constructors, do we? We just need to adjust the existing one, to now take the disableDetectScaleUnits parameter. Without a default value, so it is mandatory.

Reply:
Even if we do not have a constructor with (gravity, scale) params, it will generate the constructor in Generated/Models/AssetConversionConfiguration.cs since gravity and scale is required.

Reply:
We will still need AssetConversionConfiguration(Vector3 gravityWrapper, float scale) otherwise it will generate one in Generated/Models/AssetConversionConfiguration.cs but with public modifier and reports an error.

Reply:
I'm confused why we need an override with gravityWrapper. You are not changing any code related to gravity. Why we need an override for System.Numerics.Vector3, and one for just Vector3?